Sources/StreamChatUI/ChatChannelList/ChatChannelListItemSkeletonView.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  ChatChannelListItemSkeletonView.swift
+//  StreamChat
+//
+//  Created by Hugo Bernal on 18/07/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatChannelListItemSkeletonView: _View, ThemeProvider, SkeletonLoadable {
+
+    open private(set) lazy var avatarView: ChatChannelAvatarView = components
+        .channelAvatarView
+        .init()
+        .withoutAutoresizingMaskConstraints
+
+    open private(set) lazy var titleLabel: UILabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+
+    open private(set) lazy var subtitleLabel: UILabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+
+    open private(set) lazy var timestampLabel: UILabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+    
+    open private(set) lazy var mainContainer: ContainerStackView = ContainerStackView()
+        .withoutAutoresizingMaskConstraints
+    
+    open private(set) lazy var rightContainer: ContainerStackView = ContainerStackView(
+        axis: .vertical,
+        spacing: 4
+    )
+    .withoutAutoresizingMaskConstraints
+
+    open private(set) lazy var topContainer: ContainerStackView = ContainerStackView()
+        .withoutAutoresizingMaskConstraints
+
+    open private(set) lazy var bottomContainer: ContainerStackView = ContainerStackView(alignment: .center, spacing: 8)
+        .withoutAutoresizingMaskConstraints
```

Comment:
Would be nice to provide documentation for this view. Maybe the documentation in the class is enough 👍

Reply:
Added